recipes/samurai/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "samurai" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hpc-maths/samurai/archive/v{{ version }}.tar.gz
+  sha256: e0459f3e881b64d2e4bd99dcc5d2167265620616bf5a6e51db192f321a1a3b56
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - ninja  # [win]
+  host:
+    - xtensor
+    - highfive
+    - fmt
+    - pugixml
+    - cli11
+    - h5py
+  run:
+    - xtensor
+    - highfive
+    - fmt
+    - pugixml
+    - cli11
+    - h5py
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/samurai  # [unix]
+    - test -f ${PREFIX}/include/samurai/field.hpp  # [unix]
+    - test -f ${PREFIX}/share/samurai/samuraiConfig.cmake  # [unix]
+    - test -f ${PREFIX}/share/samurai/samuraiConfigVersion.cmake  # [unix]
+    - if not exist %LIBRARY_PREFIX%\include\samurai\field.hpp (exit 1)  # [win]
+    - if not exist %LIBRARY_PREFIX%\share\samurai\samuraiConfig.cmake (exit 1)  # [win]
+    - if not exist %LIBRARY_PREFIX%\share\samurai\samuraiConfigVersion.cmake (exit 1)  # [win]
+
+about:
+  home: https://github.com/hpc-maths/samurai
+  summary: 'Intervals coupled with algebra of set to handle adaptive mesh refinement and operators on it'
+  description: |
+    samurai allows to implement several mesh adaptation methods (patch-based, cell-based, point-based)
+    from the same data structure. The mesh is represented as intervals and a set algebra allows to efficiently
+    search for subsets among these intervals. Samurai also offers a flexible and pleasant interface to easily
+    implement numerical methods.
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
you can remove license_family since we use SPDX identifiers now and that's enough